truffle/src/main/java/org/jruby/truffle/core/thread/ThreadNodes.java

```diff
@@ -425,7 +440,8 @@ public DynamicObject allocate(
                     new AtomicReference<>(null),
                     new AtomicReference<>(null),
                     new AtomicBoolean(false),
-                    new AtomicInteger(0));
+                    new AtomicInteger(0),
+                    currentGroup);
```

Comment:
I tried to set the group in `Thread#initialize` method but I don't think `initialize` method was being called. Is allocate okay to set this for now?